### PR TITLE
Update case importance for OpenSCAP and HTTP Proxy tests

### PR DESCRIPTION
**Pull request by jameerpathan111**
This PR updates case importance for OpenSCAP and HTTP Proxy tests. It also adds 2 stubbed tests related to HTTP Proxy.

### CODE REVIEW

File: tests/foreman/cli/test_http_proxy.py [hunk 1]
```diff
@@ -88,3 +88,32 @@ def test_positive_create_update_delete(module_org, module_location):
...
+    """Verify that insights-client registration work with http proxy.
+
+    :id: 5158d5c1-2b88-4c05-914b-f1c53656ffc2
+
+    :customerscenario: true
+
+    :Steps:
+        1. Create HTTP Proxy.
+        2. Set created proxy as "Default HTTP Proxy" in settings.
+        3. Edit /etc/resolv.conf and comment out all entries so that
```
Comment (swadeley) [inline tests/foreman/cli/test_http_proxy.py:106]: Don't forget to take measures to stop NetworkManger from changing that.

File: tests/foreman/ui/test_http_proxy.py [hunk 2]
```diff
@@ -293,3 +294,34 @@ def test_check_http_proxy_value_repository_details(
...
+
+    :BZ: 1844840
+
+    :expectedresults:
+        1. "Test connection" button workes as expected.
+        2. Manifest refresh, repository enable/disable and repository sync operation
+            finished successfully.
+
+    :CaseImportance: High
+    """
```
Comment (swadeley) [inline tests/foreman/ui/test_http_proxy.py:327]: Stubbed tests need :CaseAutomation: NotAutomated at the end of the docs string

**Review by swadeley — APPROVED** [review]
ACK, with comments

File: tests/foreman/cli/test_http_proxy.py [hunk 3]
```diff
@@ -88,3 +88,32 @@ def test_positive_create_update_delete(module_org, module_location):
...
+        6. Try insights-client register/unregister/test-connection/status
+
+    :BZ: 1959932
+
+    :expectedresults:
+        1. insights-client register/unregister/test-connection/status
+            works with http proxy set.
+
+    :CaseImportance: High
+    """
```
[the remark this answers is not in the document]
Comment (jameerpathan111) [reply tests/foreman/cli/test_http_proxy.py:119]: Good catch. I forgot to add `CaseAutomation`.

File: tests/foreman/ui/test_http_proxy.py (hunk 2 above)
Comment (jameerpathan111) [reply tests/foreman/ui/test_http_proxy.py:329]: updated the pr.

**Review by omkarkhatavkar — APPROVED** [review]

**Pull request merged**